app/src/main/res/layout/list_item_location.xml

```diff
@@ -16,7 +16,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:gravity="start"
-            android:text="06/24/2016\n16:54:32"
+            android:text="@string/_06_24_2016_n16_54_32"
```

Comment:
This string is just a mockup, it doesn't need to be translated. Please either leave it as it was or delete this attribute altogether.